OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -394,7 +394,7 @@ Action InitInputPanel(Widget panel)
 					{ "CycleBaseKey", "Jump to base" },
 					{ "ToLastEventKey", "Jump to last radar event" },
 					{ "ToSelectionKey", "Jump to selection" },
-					{ "SelectAllUnitsKey", "Select all units on screen" },
+					{ "SelectAllUnitsKey", "Select all military units" },
```

Comment:
The context is that it's no longer just "on screen" but alternates between "on screen" and "across the entire map".